cordova-lib/spec-cordova/ConfigParser.spec.js

```diff
@@ -114,78 +114,94 @@ describe('config.xml parser', function () {
                 expect(cfg.getPlatformPreference('orientation', 'foobar')).toEqual('');
             });
         });
-        describe('feature',function(){
-            it('should read feature id list', function() {
+        describe('plugin',function(){
+            it('should read plugin id list', function() {
                var expectedList = [
                    'org.apache.cordova.featurewithvars',
```

Comment:
since we are changing names from 'feature' to 'plugin', these ids should be updated. instead of 'org.apache.cordova.featurewithvars', please use 'org.apache.cordova.pluginwithvars'.